src/PersonObjects/Sleeve/Sleeve.ts

```diff
@@ -196,24 +194,24 @@ export class Sleeve extends Person {
     cyclesUsed = Math.min(cyclesUsed, 15);
     this.shock = Math.min(100, this.shock + 0.0001 * cyclesUsed);
     if (!this.currentWork) return;
-    this.currentWork.process(p, this, cyclesUsed);
+    this.currentWork.process(Player, this, cyclesUsed);
```

Comment:
Player can also be removed as a parameter from sleeve work functions, but those are defined in different files. I kept the parameter changes in this PR basically to just Sleeve.ts to increase chances this gets merged.